clients/python/llmengine/data_types.py

```diff
@@ -152,20 +156,28 @@ class GetLLMEndpointResponse(BaseModel):
     Response object for retrieving a Model.
     """
 
-    id: str = Field(description="The autogenerated ID of the Launch endpoint.")
-    """The autogenerated ID of the Launch endpoint."""
-    name: str = Field(description="The name of the Launch endpoint.")
-    """The name of the Launch endpoint."""
-    model_name: str = Field(description="The name of the model.")
-    """The name of the model."""
-    source: LLMSource = Field(description="The source of the model.")
-    """The source of the model."""
+    id: Optional[str] = Field(default=None, description="(For self-hosted users) The autogenerated ID of the model.")
+    """(For self-hosted users) The autogenerated ID of the model."""
```

Comment:
nit: can we add a newline after each of these docstrings to let things visually breathe?